shuup/front/templates/shuup/front/macros/basket.jinja

```diff
@@ -1,4 +1,5 @@
 {% extends "shuup/front/macros/theme/basket.jinja" %}
+{% from "shuup/front/macros/order.jinja" import render_package_children %}
```

Comment:
Insteadn of importing this you should be able to use `{{ order_macros.render_package_children(product) }}` where every the render_package_children is used.